osu.Game/Skinning/ArgonProSkin.cs

```diff
@@ -24,9 +24,11 @@ public class ArgonProSkin : ArgonSkin
         {
             foreach (string lookup in sampleInfo.LookupNames)
             {
-                string remappedLookup = lookup.Replace(@"Gameplay/", @"Gameplay/ArgonPro/");
+                var sample = Samples?.Get(lookup)
+                             ?? Resources.AudioManager?.Samples.Get(lookup.Replace(@"Gameplay/", @"Gameplay/ArgonPro/"))
+                             ?? Resources.AudioManager?.Samples.Get(lookup.Replace(@"Gameplay/", @"Gameplay/Argon/"))
+                             ?? Resources.AudioManager?.Samples.Get(lookup);
```

Comment:
I meant to double-check on this. I'm not sure the `Resources.AudioManager` calls should be required.

Reply:
I think this is a much-of-a-muchness at this point, until we start to solidify how skinning should work with these new skins. It doesn't change behaviour because there are no internal usages that rely on the initial Sample?.Get() lookup path.